src/components/Header/Header.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export default function Header() {
+  return (
+    <div>
+      <ul>
+        <li>
+          <NavLink exact to='/' style={{ textDecoration: 'none' }}>
+            Home
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to='/teams' style={{ textDecoration: 'none' }}>
+            Teams
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to='/players' style={{ textDecoration: 'none' }}>
+            Players
+          </NavLink>
+        </li>
+      </ul>
```

Comment:
To prevent the bullet points from rendering, we can wrap the navlinks in a div or section tag to make styling easier.